components/base/kinematicbase/kinematics.go

```diff
@@ -23,17 +24,17 @@ type KinematicBase interface {
 
 const (
 	// LinearVelocityMMPerSec is the linear velocity the base will drive at in mm/s.
-	defaultLinearVelocityMMPerSec = 200
+	defaultLinearVelocityMMPerSec = 200.
```

Comment:
All of these are meant to be floats and are already used as such. Adding the `.` to the end makes it clear that they are, rather than leaving it ambiguous and leaving it to the compiler to figure it out based on usage.